core/vibes/avios/sections/cart/coupon-code-form/index.tsx

```diff
@@ -1,13 +1,13 @@
 'use client';
 
+import { Button } from '@alto-avios/alto-ui';
```

Comment:
Same for this import '~/alto/alto-avios'
```suggestion
import { Button } from '~/alto/alto-avios';
```